src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs

```diff
@@ -33,7 +33,7 @@
        :toolbar
        (fn [{:keys [reorderer]}]
          [:div {:style {:display "flex" :alignItems "top"}}
-          [:div {:style {:fontWeight 700 :fontSize "125%"}} "Search Results: "
+          [:div {:style {:fontWeight 700 :fontSize 20 :marginBottom "1em"}} "Search Results: "
```

Comment:
Absolute sizes always make me a bit squidgy, so consider representing this as 125%. Not the end of the world, though.